components/aria2c/Dockerfile

```diff
@@ -0,0 +1,42 @@
+# Pulling ubuntu bionic as base image
+FROM ubuntu:18.04
+
+# defining dockerfile maintainer
+MAINTAINER SCoRe Lab Community <commuity@scorelab.org>
+
+ARG BUILD_DATE
+ARG VCS_REF
+
+# Setting current work directory to /home/Bassa/aria2
+WORKDIR /home/Bassa/aria2
```

Comment:
I think we don't need this as the `WORKDIR` we need is  `/data` and it is defined below.

Reply:
@agentmilindu You are right Bro!! Will remove that

Reply:
@agentmilindu are there any more fixes Please let me know bro so that I can make all these changes in the upcoming commit. 😄

Reply:
@agentmilindu fixed :)